rakelib/check.rake

```diff
@@ -9,8 +9,11 @@ namespace :check do
     path = ENV['path']
     unless path
       modified_files = `git ls-files --modified --others --exclude-standard`.split("\n").reject {|file| File.extname(file) == '.md'}
```

Comment:
no need to reject md files here, because the image-only files will be selected on line 14.
see devdocs implementation

Reply:
the line should've been changed like [here](https://github.com/magento/devdocs/blob/master/rakelib/check.rake#L12) to
```
modified_files = `git ls-files --modified --others --exclude-standard`.split("\n")
```

now when you run `rake check` it reports about a missing variable

```
~/Projects/repos/magento/merchdocs (db_rake_updates)$ rake check:image_optim
rake aborted!
NameError: undefined local variable or method `modified_files' for main:Object
```